lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -453,7 +491,6 @@ defmodule RealtimeWeb.RealtimeChannel do
   end
 
   def handle_in(_, _, socket) do
-    socket = count(socket)
```

Comment:
Again, make sure we're handling all messages and remove the whole function.

Reply:
yep, removing